api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -117,9 +118,14 @@ public void setUp() {
 
     mockDeletedWorkspace =
         buildDbWorkspace(2, "workspace_del", "workspaceNs", WorkspaceActiveStatus.DELETED);
-
+    mockDeletedWorkspace.setCreator(dbUserWithEmail);
     when(mockWorkspaceDao.findDbWorkspaceByWorkspaceId(2)).thenReturn(mockDeletedWorkspace);
 
+    mockCreatorWorkspace =
+        buildDbWorkspace(3, "mock_workspace_name", "workspaceNS", WorkspaceActiveStatus.ACTIVE);
+    mockCreatorWorkspace.setCreator(dbUserWithoutEmail);
```

Comment:
I would have expected the creator to be a field passed in to `buildDbWorkspace`.